src/Diagnostics/atmos_les_default.jl

```diff
@@ -577,14 +511,32 @@ function atmos_les_default_collect(dgngrp::DiagnosticsGroup, currtime)
         s -> startswith(s, "moisture.") ? s[10:end] : s,
         flattenednames(vars_atmos_les_default_simple(bl, FT)),
     )
-    for vari in 1:length(simple_varnames)
-        for evk in 1:(Nqk * nvertelem)
-            simple_ha = atmos_les_default_simple_vars(bl, simple_avgs[evk])
-            avg_rho = simple_ha.avg_rho
+    for evk in 1:(Nqk * nvertelem)
+        simple_ha = atmos_les_default_simple_vars(bl, simple_avgs[evk])
+        avg_rho = simple_ha.avg_rho
+        for vari in 1:length(simple_varnames)
             if simple_varnames[vari] != "avg_rho"
                 simple_avgs[evk][vari] /= avg_rho
             end
         end
+
+        # for LWP
+        # FIXME properly
+        if isa(bl.moisture, EquilMoist) || isa(bl.moisture, NonEquilMoist)
+            ρq_liq_z[evk] /= avg_rho
+        end
+    end
+
+    # compute LWP
+    lwp = NaN
+    if mpirank == 0
+        JcV = reshape(
+            view(vgeo, :, grid.JcVid, :),
+            Nq^2,
+            Nqk * nvertelem,
+            nhorzelem,
+        )
+        lwp = FT(sum(ρq_liq_z .* JcV[1, :, 1]))
```

Comment:
@jkozdon: does this look correct?

Reply:
The `reshape` is throwing an error [here](https://github.com/CliMA/ClimateMachine.jl/pull/1549/checks?check_run_id=1114577318#step:7:309). :-(

Reply:
You need to mask out the ghost elements in the reshape
```suggestion
        JcV = reshape(
            view(vgeo, :, grid.JcVid, topology.realelems),
            Nq^2,
            Nqk * nvertelem,
            nhorzelem,
        )
        lwp = FT(sum(ρq_liq_z .* JcV[1, :, 1]))
```